Package.swift

```diff
@@ -1,8 +0,0 @@
-import PackageDescription
```

Comment:
I just deleted this file since SPM can not be used for iOS yet. I'll add it again in the future.